packages/sanity/src/core/releases/components/dialog/DiscardVersionDialog.tsx

```diff
@@ -30,6 +32,11 @@ export function DiscardVersionDialog(props: {
   const schema = useSchema()
   const toast = useToast()
   const [isDiscarding, setIsDiscarding] = useState(false)
+  const discardType = isDraftId(documentId) ? 'draft' : 'release'
+  const releaseName =
```

Comment:
Might you be able to try just
```suggestion
  const releaseName = isDraftOrPublished(discardFromPerspective) ? discardFromPerspective : discardFromPerspsective.metadata.title
```

Reply:
I don't think so since this is mostly applied to releases and not to documentIds. I see that it's not being used anywhere so we might need to remove it. 
In any case I would keep it as is since we will never discard a Publish "version" this way anyway :)